examples/src/main/scala/org/apache/spark/examples/ml/QuantileDiscretizerExample.scala

```diff
@@ -38,6 +38,7 @@ object QuantileDiscretizerExample {
       .setInputCol("hour")
       .setOutputCol("result")
       .setNumBuckets(3)
+      .setRelativeError(0)
```

Comment:
If we do this here, then I think we should be explicit about the fact that we're computing exact quantiles for illustrative purposes.

Something like 
```
.setRelativeError(0) // note that we compute exact quantiles here for illustrative purposes, however in most cases the default parameter value should suffice 
```